os/board/rtl8730e/src/component/soc/amebad2/fwlib/ram_hp/Make.defs

```diff
@@ -51,8 +51,12 @@
 
 CSRCS += ameba_sport.c \
 	ameba_audio_clock.c \
-	ameba_system.c
-
+	ameba_system.c \
+	ameba_lcdc.c \
+	ameba_mipi.c
+#ifdef CONFIG_EXAMPLES_MIPI
+CSRCS += ameba_mipi_show.c 	 
+#endif
```

Comment:
We don't use the MIPI always. Could you provide us some new config to include and to exclude for flash usage?

Reply:
Dear @sunghan-chang  
Okay, i will make the relevant changes

Reply:
Dear @sunghan-chang,
I missed out the changes in the previous PR.. Sorry for the confusion. 
May i check if its ok to build the example in bsp (os/board), and call from board_specific_demo using extern? Like what i have explained earlier, this mipi example is calling our driver APIs directly, currently there is no porting layer/framework to link application layer to bsp layer yet, so moving ameba_mipi_show.c and ameba_mipi_display.h into app/example will create a lot of compile errors due to missing headers..
Thank you

Reply:
@zhongnuo-tang Let's keep it in the board as you did.